### PR TITLE
Support for s3-large-x86

### PR DESCRIPTION
**Pull request by majst01**
With this we have:
- consolidated lldp server to support newer intel 100g nics
- remove encryption of ssd´s because this will require bios password for some ssd vendors
- wipe of disks migrated from waitgroup to errgroup
- new disk layout for s3-large-x86 size, added ability to create a spare lvm partition

TODO:
- [ ] fixate go-lldpd version once released

tested already

### CODE REVIEW

File: cmd/reinstall.go [hunk 1]
```diff
@@ -33,6 +33,7 @@ func (h *Hammer) reinstall(m *models.ModelsV1MachineResponse, hw *models.DomainM
 		return false, errors.New("machine is not yet ready for reinstallations, too risky to wipe disks")
 	}
 	var currentPrimaryDiskName string
+	var err error
```
Comment (majst01) [inline cmd/reinstall.go:36]: Why

File: cmd/storage/wipe.go [hunk 2]
```diff
@@ -35,28 +31,31 @@ func WipeDisks() error {
...
+		if ok && strings.Contains(disktype, "isw_raid") {
+			log.Info("skip raid member", "disk", disk.Name)
+			continue
+		}
+
+		g.Go(func() error {
 			if strings.HasPrefix(disk.Name, DiskPrefixToIgnore) {
-				return
+				return nil
 			}
```
Comment (Gerrit91) [inline cmd/storage/wipe.go:50]: This can also be moved outside the go routine like the other skip condition.

File: cmd/storage/wipe.go [hunk 3]
```diff
@@ -35,28 +31,31 @@ func WipeDisks() error {
...
+		disk := disk
+		properties, err := FetchBlockIDProperties(fmt.Sprintf("/dev/%s", disk.Name))
+		if err != nil {
+			log.Warn("failed to detect disk properties", "error", err)
+		}
+		disktype, ok := properties["TYPE"]
+		if ok && strings.Contains(disktype, "isw_raid") {
+			log.Info("skip raid member", "disk", disk.Name)
+			continue
+		}
```
Comment (Gerrit91) [inline cmd/storage/wipe.go:45]: I think just for safety with accessing `nil` maps, I'd put this into an `else` branch.

File: cmd/storage/wipe.go (hunk 2 above)
Comment (Gerrit91) [reply cmd/storage/wipe.go:50]: Maybe also emit a log message that it skipped because its prefix is ignored for wipe.

**Review by Gerrit91 — APPROVED** [review]

**Pull request merged**